### PR TITLE
feat : Added Banner and Category component

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added Banner  and Category component in Home page 

- Category shown are from backend 
 
## what to check 

- Banner.jsx -src/components/banner/Banner.jsx
- BannerOffer.jsx - src/components/banner/BannerOffer.jsx
- Category.jsx - src/components/category/Category.css
- Styles Sheets

Screenshots
storeBanner
storeBannerOffer
storeCategory

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
Code is written well structured. 
The whole code is broken into small components which is like a lot.
And your UI looks amazing. 
Good to merge

File: src/components/banner/BannerOffer.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
+
+function BannerOffer(){
+
+    return<>
+
```
Comment (JantuDeb) [inline src/components/banner/BannerOffer.jsx:7]: div is the only child here so you can remove this empty `<>` `</>`

File: src/components/category/Category.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+function Category(){
+
+const [category , setCategory ] = useState([]);
+
+ useEffect(()=>{
+
+    (async()=>{
+
+        try {
+            const response = await axios.get("/api/categories");
```
Comment (JantuDeb) [inline src/components/category/Category.jsx:17]: const {data} = await axios...

**Review by JantuDeb — APPROVED** [review]
Added a suggestion

**Review by neetukumari4858 — APPROVED** [review]
overall looks good.code is readable that i like.

File: src/components/banner/BannerOffer.jsx (hunk 1 above)
Comment (sushil-mod) [reply src/components/banner/BannerOffer.jsx:7]: yes , thank you

File: src/components/category/Category.jsx (hunk 2 above)
Comment (sushil-mod) [reply src/components/category/Category.jsx:17]: yeah , thanks for suggestion i will use this

**Pull request merged**